gui/ThemeEngine.cpp

```diff
@@ -919,7 +919,7 @@ void ThemeEngine::drawDDText(TextData type, TextColor color, const Common::Rect
 		restoreBackground(dirty);
 
 	_vectorRenderer->setFgColor(_textColors[color]->r, _textColors[color]->g, _textColors[color]->b);
-	_vectorRenderer->drawString(_texts[type]->_fontPtr, text, area, alignH, alignV, deltax, ellipsis, dirty);
+	_vectorRenderer->drawString(_texts[type]->_fontPtr, TransMan.bidiAlgo(text), area, alignH, alignV, deltax, ellipsis, dirty);
```

Comment:
This will break the compilation when translations is disabled (i.e. if `USE_TRANSLATION` is not defined).

Reply:
Done.
I've also updated the initial PR description a little to much the updated compilation instructions.